ui/src/containers/Header/index.tsx

```diff
@@ -103,10 +104,32 @@ const Header: React.FC = observer(() => {
         className="hub-header-login__modal"
         aria-label="login"
       >
-        <TextContent>
-          <Divider />
-          <Text component={TextVariants.h6}>Sign In With</Text>
-        </TextContent>
+        {providers.items.size !== 0 ? (
+          <TextContent>
+            <Divider />
+            <Text component={TextVariants.h6}>Sign In With</Text>
+          </TextContent>
+        ) : (
+          <div>
+            <Divider />
+            <Grid>
+              <GridItem span={1}>
+                <Grid>
+                  <GridItem offset={11} className="hub-header-sigin-modal-icon">
+                    <Icon id={Icons.InfoCircleIcon} size={IconSize.sm} label={'info'} />
+                  </GridItem>
+                </Grid>
+              </GridItem>
+              <GridItem offset={1}>
+                <TextContent>
+                  <Text component={TextVariants.h6}>
+                    No oauth providers configured. Please contact the owners
```

Comment:
Instead of hard coding this can we add this message in the constant errors define in common directory ?